testing/upload_validation.ts

```diff
@@ -91,6 +91,12 @@ import * as z from 'zod';
  * a real RegExp object.
  */
 export const PACKS_VALID_COLUMN_FORMAT_MATCHER_REGEX = /^\/(.*)\/([a-z]+)?$/;
+export const BUILDING_BLOCK_COUNT_LIMIT = 500;
+export const BUILDING_BLOCK_NAME_CHARACTER_LIMIT = 50;
```

Comment:
You could maybe group these together like

```typescript
const Limits = {
  BuildingBlockDescription: 500,
  ColumnMatcherRegex: 100,
  NumColumnMatchers: 10,
};
```

Better namespaced, and makes your imports cleaner too.